bigint/src/lib.rs

```diff
@@ -3469,23 +3547,20 @@ mod biguint_tests {
 
         check(BigUint::new(vec![]), 0);
         check(BigUint::new(vec![1]), (1 << (0 * big_digit::BITS)));
-        check(BigUint::new(vec![N1]), (1 << (1 * big_digit::BITS)) - 1);
-        check(BigUint::new(vec![0, 1]), (1 << (1 * big_digit::BITS)));
-        check(BigUint::new(vec![N1, N1 >> 1]), i64::MAX);
+        check((BigUint::one() << 63) - BigUint::one(), i64::MAX);
```

Comment:
These tests shouldn't be changed.  Whatever people might give on the public interfaces needs to keep working!